packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -291,6 +294,25 @@ ${bodyDetail}`
       if (context.payload.pull_request === undefined) {
         return;
       }
+      if (context.payload.action === 'labeled') {
+        // Only proceeds if `snippet-bot:refresh` label is added.
+        if (context.payload.pull_request.labels === undefined) {
+          return;
+        }
+        let labelFound = false;
+        for (const label of context.payload.pull_request.labels) {
```

Comment:
you could use ` ["banana", "apple"].find((item) => { return item === 'banana' })` here.

Reply:
I changed it to use `some`, but I'm not sure if it became simpler.